src/common-utils/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "common-utils",
-    "version": "2.4.5",
+    "version": "2.4.6",
```

Comment:
Looks like we have multiple test failures due to this PR change, can you help fix those? 

https://github.com/devcontainers/features/actions/runs/10196886412/job/28208566441?pr=1069#step:4:1167

Reply:
Done. resolved